Sources/MapboxMaps/Location/Puck/Puck2D.swift

```diff
@@ -313,3 +346,9 @@ private extension Puck2DConfiguration.Pulsing.Radius {
         }
     }
 }
+
+private extension ClosedRange where Bound == Double {
```

Comment:
nit: can we have a little more generic constraints and not just Double only
```suggestion
private extension ClosedRange where Bound: AdditiveArithmetic {
```

Reply:
Nice tip, I got lost in Swift's numerics :)